bin/rucio

```diff
@@ -1805,23 +1806,26 @@ def add_lifetime_exception(args):
         scope, name = get_scope(did, client)
         dids_list.append({'scope': scope, 'name': name})
     summary = {0: len(dids_list), 1: 0, 2: 0, 3: 0, 4: 0}
-    for meta in client.get_metadata_bulk(dids_list):
-        scope, name = meta['scope'], meta['name']
-        dids_list.remove({'scope': scope, 'name': name})
-        if meta['did_type'] == 'FILE':
-            logger.warning('%s:%s is a file. Will be ignored' % (scope, name))
-            summary[1] += 1
-        elif meta['did_type'] == 'CONTAINER':
-            logger.warning('%s:%s is a container. It needs to be resolved' % (scope, name))
-            containers.append({'scope': scope, 'name': name})
-            summary[2] += 1
-        elif not meta['eol_at']:
-            logger.warning('%s:%s is not affected by the lifetime model' % (scope, name))
-            summary[3] += 1
-        else:
-            logger.info('%s:%s will be declared' % (scope, name))
-            datasets.append({'scope': scope, 'name': name})
-            summary[4] += 1
+    chunk_limit = 500  # Server should be able to accept 1000
```

Comment:
Can't we take the `maxItems` entry (or some fraction of it) from one schema (e.g. atlas)?
https://github.com/rucio/rucio/blob/47e93cfbe5887071c70de4ba815c1bbdddfac2ce/lib/rucio/common/schema/atlas.py#L259

This reduces the number of hard-coded constants in our code by one.

Reply:
The problem is that the schema is not necessarily part of the client, so this information might not be available to the client.